index.js

```diff
@@ -1001,7 +1001,10 @@ Feed.prototype.close = function (cb) {
   this._ready(function () {
     self.writable = false
     self.readable = false
-    self._storage.close(cb)
+    self._storage.close(function (err) {
+      self.emit('close')
```

Comment:
should only be emitted if no error occured during storage.close. Also mind adding a flag '.closed` or similar so we don't emit this twice?